pkg/kgo/record_and_fetch.go

```diff
@@ -44,7 +44,7 @@ func (a RecordAttrs) TimestampType() int8 {
 	if a.attrs&0b1000_0000 != 0 {
 		return -1
 	}
-	return int8(a.attrs & 0b0000_1000)
+	return int8((a.attrs & 0b0000_1000)) >> 3
```

Comment:
Unnecessary double parens, mind removing and squashing into one commit? I could too if you don't get to it before I do (though I'm working my way through #210 so I won't be getting to this _that_ fast)

Reply:
Squashed. :)